src/server/js/resources/health/schema.js

```diff
@@ -1,26 +1,10 @@
 import HealthModel from './model';
 
 const typeDefs = `
-    type HealthModel {
+    type Health {
```

Comment:
can ignore the `health`-related files, i will comment on the convention i've (finally) settled on using the houses model as an example below